src/test/c_api_tests.cpp

```diff
@@ -236,10 +237,79 @@ TEST_F(CapiInferencePreparationTest, Basic) {
     //////////////////
     //  INFERENCE
     //////////////////
+    // we need to prepare server now TODO
+    // remove when C-API start implemented
+    std::string port = "9000";
+    randomizePort(port);
+    char* argv[] = {
+        (char*)"OpenVINO Model Server",
+        (char*)"--model_name",
+        (char*)"dummy",
+        (char*)"--model_path",
+        (char*)"/ovms/src/test/dummy",
+        (char*)"--port",
+        (char*)port.c_str(),
+        (char*)"--log_level",
+        (char*)"DEBUG",
+        nullptr};
+
+    ovms::Server& server = ovms::Server::instance();
+    std::thread t([&argv, &server]() {
+        ASSERT_EQ(EXIT_SUCCESS, server.start(9, argv));
+    });
+    auto start = std::chrono::high_resolution_clock::now();
+    while ((ovms::Server::instance().getModuleState("GRPCServerModule") != ovms::ModuleState::INITIALIZED) &&
+           (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::high_resolution_clock::now() - start).count() < 1)) {
+    }
+    std::this_thread::sleep_for(std::chrono::seconds(1));  // TODO
+    OVMS_InferenceResponse* response = nullptr;
+    OVMS_Server* cserver = reinterpret_cast<OVMS_Server*>(&server);
+    status = OVMS_Inference(cserver, request, &response);
+    ASSERT_EQ(nullptr, status);
+    // verify GetOutputCount
+    uint32_t outputCount = -1;
```

Comment:
I'm pretty sure klocwork will flag it as overflow

Reply:
Use limits max uint32?

Reply:
Or just any other number than 1

Reply:
I don't think we scan tests with KW - there would be much bigger issues there.